server/index.js

```diff
@@ -2,17 +2,49 @@ const express = require("express");
 const bodyParser = require("body-parser");
 require("dotenv").config();
 const cors = require("cors");
+const yaml = require("js-yaml");
 
-require("./cron-jobs/processCompletedInterviews");
 
+// services
+
+let emailServiceEnabled;
+try {
+  const configPath = path.join(__dirname, "config/dev.config.yaml");
+  const config = yaml.load(fs.readFileSync(configPath, "utf8"));
+  emailServiceEnabled = config.emailService;  // Extract emailService flag
+} catch (error) {
+  console.error("Error loading YAML config:", error);
+  process.exit(1);
+}
+let openAIServiceEnabled;
+try {
+  const configPath = path.join(__dirname, "config/dev.config.yaml");
+  const config = yaml.load(fs.readFileSync(configPath, "utf8"));
+  openAIServiceEnabled = config.evaluationByGPT;  
```

Comment:
Load the file once, and try reading all the variables from the config. No need to load the file for every variable